### ISSUE TITLE
[Improvement] Allow tooltip component in a row component

### ISSUE THREAD

**Issue by KillianH** [MetaMask/snaps#2972]
I'll like to have tooltips in rows to display full value of long values.
This isn't supported atm.

**Comment by FrederikBolding** [comment]
Thanks for the suggestion! We'll look into this, but in the meantime `Row` has a `tooltip` property that applies a tooltip to the left side of the row.

**Comment by KillianH** [comment]
Yep, but it's a little bit confusing for the user to display a tooltip here for the value on the right.
